server/src/main/java/io/crate/metadata/settings/CoordinatorSessionSettings.java

```diff
@@ -42,7 +42,7 @@ public class CoordinatorSessionSettings extends SessionSettings {
 
     private final User authenticatedUser;
     private User sessionUser;
-    private Set<Class<? extends Rule<?>>> excludedOptimizerRules;
+    private Set<Class<? extends Rule>> excludedOptimizerRules;
```

Comment:
`rule.getClass()`only returns `Class<? extends Rule>` instead of `Class<? extends Rule<?>>` without a cast. This makes no difference.